src/RadPulse/test_radiation_pulse.cpp

```diff
@@ -30,6 +31,7 @@ template <> struct quokka::EOS_Traits<PulseProblem> {
 	static constexpr double mean_molecular_weight = 1.0;
 	static constexpr double boltzmann_constant = (2. / 3.);
 	static constexpr double gamma = 5. / 3.;
+	static constexpr double hydrogen_mass_code_units = quokka::hydrogen_mass_cgs;
```

Comment:
It should be possible to fix the test failure by changing to the atomic mass unit instead.